### PR TITLE
modified slack integration as new app

### PR DESCRIPTION
**Pull request by himorin**

### CODE REVIEW

File: development-management/request.rst [hunk 1]
```diff
@@ -84,7 +84,8 @@ as naming conventions, following procedures shall be performed by sysadmin.
 3. Add a line of new repository to `repos.html <https://pfspipe.ipmu.jp/repos.html>`_ and `about.html <https://pfspipe.ipmu.jp/about.html>`_.
 4. Create new JIRA project or component, if required.
 5. Check JIRA integration to load repository updates (branch, PR etc.).
-6. Modify github-slack integration to add repository to channel via `slack app management page <https://sumire-pfs.slack.com/apps/manage>`_.
+6. Add target repository by command '/github subscribe Subaru-PFS/<repo_name>' 
+   at #github channel of PFS slack.
```
Comment (yukimoritani) [inline development-management/request.rst]: Could you just confirm us... "command" means just typing in message box?

Comment (himorin) [reply development-management/request.rst]: ah, yes.
refer: https://api.slack.com/slash-commands

Comment (yukimoritani) [reply development-management/request.rst]: I see. Thanks.

**Review by yukimoritani — APPROVED** [review]

**Comment by himorin** [comment]
added reference to slash command as link
canceled awaiting r? and merged to master.

**Pull request merged**